README.md

```diff
@@ -168,6 +168,15 @@ config :ueberauth, Ueberauth,
   ]
 ```
 
+## Customizing JSON Serializer
+
+Your json serializer can be configured depending on what you have installed in your application.  Defaults to [Jason](https://github.com/michalmuskala/jason) in ueberauth libraries such as [ueberauth_google](https://github.com/ueberauth/ueberauth_google) or [ueberauth_twitter](https://github.com/ueberauth/ueberauth_twitter).
```

Comment:
I don't think we need to add those links here, to be honest. Anyway, this package wouldn't care about those in terms of dependencies.

```suggestion
Your json serializer can be configured depending on what you have installed in your application.  Defaults to [Jason](https://github.com/michalmuskala/jason).
```